docs/src/views/Architecture.tsx

```diff
@@ -0,0 +1,213 @@
2m9IycC+CFthe18/S6y7FYGoB7xknUbvAA+434vD7OuWwtWIffkr+ALkZ9AGB3rAL8UbA7h2/RWGAOUpIFm+5tAOvIyrDxIw1e80Rgd7ISDVF4P458P1qRPijhsv7guS1YerTwy4iP7ccX18LuAMJ1krA05jTNgNXEq5jrF+clnkr4icH+KGgdxj97nmCe/78ZjGJuD3972AbWlpSMgKYq/JGUyP4ye+O8qfTzFDkqWelAvJWrRs2jJWlBQ5ksVPgM8JL1gkWeJESpLVuF3jWj/+3HQ1q2o8JFmFJ1eyuM+QnBz0aSsJyep/3Q2hdwOF4T6+PizADS7J+SVrFCcyQRkBwi0T+vpaYFlcIG48TEPoozRcTgvF6mD5lYeDrznifCa/PRM3smKxNtQd5hnRGMYmkQ8L4gTufjIi+P0PUjghs5E/hu95HM9ORfTjTBhGFrxqzxybklOQ8jRT6Df0STYu3Y3Fghts8j137PY7Mbj2MAuhXJ8nE3ywJy0BW69xkuVrCePUNERxx/E3VD0R4ywc05uTr+kRfpyopXB9ZuBqVjS2hTugH3ueU8OuwPXeXeHx/FOioR/OvSb8Hd/5xxNu4XAHq+gWDhpBuZAsDkkOF8pkH1TUO9p8z+IjP+Rm4eEmZ0myxInU5mQReZBkFZ6tdq2wy6EbDrsNg4nnBCFGbuOx22roDPYSqoySSpbWlbPYkJKCnSF5Q4aUt2dogCu8UhUmwRczJFnqSbmQrPoNPpekZClj0aEf2pNkiRMpSZZs/4Jam3cuX86qGg9JVuHZ59oLR5yntbL1W1bTL9Y0N+zlUykllyz+akBXOKbeyL0hKeUt8T2Pw/yyOoHK5nMVHpIs9aRcSNayZbN+YEXJQ5IlXmhOlnQhySo8+1y1cNx9YZl8zr2LZPHzjCYE2uUOGVLeEv4u8SIsEE2IR7mQLP1Vur1YUfKQZIkXKUlW3YZ1a9dv8BlJFoMkq/BIX7IopR2SLPWkXEhWlSqV1Wq4cLtTK+EqQ0rZZsOFlpxktZHKmawK/fo1/h8razyaLFn8BTFHr7XH4avtcci/XV78uLpvByEGTr1JsjQ8JFnqSbmQrAoVKqiNZC0zbVpT73Cz7gsON6aUcfjXnb8Ygb0VonLK7dRH5x2OTmZVjafcSxYnUrxMHfBph90ev2G7fbt/t1q3+3f1mZ/+XX+29b+7bfv9e9xj5r+Wfqtys9mqY8+N5h2689lqodXd4qpeNfZylSokWdIMSZZ6snbt2p91dHT+lVqmTp0as3LlyrrsaRbOklUzx7IiQagLNCdLgfIkWfytFvZ5t8UuVy5OnXDEfeTzjefaH9A70vzAdouBB85dXn0AkElC9pVBkiXNkGQRorFixazvWJEg1IK6devWrlGjOkkWQ10liz87ddi/PfZ7t4OBS9d/1p9pfX2HRd/r1n6bXFYYtam93VKrtq3fjlK5CrC0IMmSZkiyCNH46uv6ajNcSBA83bt3r9y++y/0fw4Y6iZZ/NmqPxx+e73VsvOrI65j960xa/P7cdcFndivo9aQZEkzJFmEmJBkEWqFpaVllQtOpr+xqsYjdck6er099nm1wy7njthi1cl84+luh1wDj8xhT79cQZIlzZBkEaIxZ9HkVaxIEGoBP1xYvToNF+YgOcnihwGv8sOA7fk7rT+Wnelw46THPB2ZWfc6MgnPp1IFJFnSDEkWISYV2E+CUBdo4rsCUpGsIwHtsdOl7etNFm1fn748//6a4+16nr+09hf2NDUCkixphiSLEA2tvp1puJBQKzp06FCtU7c2A1hV41l7urXtbudO2OXK39NMfr8oflK5MhFSZQ75tYOBR3vsce6KjRZtjdaeaH/0hOuCHuxpaSQkWdIMSRYhJiRZhFphaWlZycrF8AtW1XiMjHQ/lNl0ryMz+6XOWe/lvffajgxaefyXoG1WXRL2ufTCXvfOwpyogz6cgPnJJ57z4W/iKeR6e7k4cWKWu40Lf2aKb8cfw9/Uk+9jr3tXbLPpELXqRMugo05Tgs76LG2512ZoHfZUNB6SLGmGJIsQjW+++TqN++HBpTVf//zzT2xYfTFf//rrL3ay+mF5vcHgChUq8HV3vt7guwZNq1atwtf51OBSoU7dWjn1X7l88NMvTW35er9Bv3u4uZ36qEOX1rv5euMm33lYuR77pVPXNkMrV67oUbFiBY+LTkbzO3Zv06xWrRpCH2bnDQ5qa2tX+q7RV0J9wx/6btxPrs8mdnx9wOCeHqdO7fqofefWBny9SdOGHhfsTFp27tZqRMWKXJ9cvxedTea079aqeU3W5wmLvQf4K9S+ayjvc+P2ZS7czw9++rmJPV8fNEzLw97eqHrHTq328vUmzRp6WDmY/tSxS5tRfJ+VK1f2sHQ0mtWly68tatZkfZ7ft79NmzYffvvdl0J9885lztzP3D6HjuzjYWGxp1qHTq328/WmLRp52Dofa9Gpa6sxfJ8ffvihh5WTyYzREwf+9PEndYU+/jBYtbdfv37/a9mqmVBfsGQaf/fbD3podXLg61p9OnuYmcmqjhzT/wBfb9joa4+9h9c2154weBz/evJ97j0imzZq/NCfP/64jtDHtr0rDXR0ulfl3hOhvmj5dL6vD3r06uDI13v37+Yhk8mqjhjd7xBfb9T4W66PTU25PifyfVapwvV5VDZl7OTBLeuyPrnnuWfUqFHVfmyZ0+cM/nfO7bPPgN899u9f8L/ho/oe4ev8+777oKwJ97tO5n93/u/H4MiGSdrjB/xal/3tbN+7ereu7qDqP/7URKgvXjGDf78/6N6zgxP3+nt17NyahguLwOP6wY8drxpoGdiO1lpq9KvWymO/asnOd9A677v833M+S7HHrr+QP6y6Y/XpH7Hu7C/Ybdsnd/sh51Gw8F2JLRe79pWZd+D6+FnriONULbewXR+xhyAK0PbQ+v/ammwGRWIx3IgOJFlEmfPBB/8HSefvMf4S+NwAAAAASUVORK5CYII=" />
```

Comment:
it is better to introduce this image to the repo and reference it by path - otherwise it is impossible to review just the image's content

Reply:
done